knowledge-base/kb-gridview-howto-change-cells-tooltip-duration.md

```diff
@@ -0,0 +1,42 @@
+---
+title: How to set the cell's tooltip duration 
+description: Control the duration of a cell's tooltip
+type: howto
+page_title: Change the timespan that a cell's tooltip is shown 
+slug: kb-gridview-howto-change-cells-tooltip-duration
+position: 0
+tags: cell, tooltip, duration, timespan
+ticketid: 1448661
+res_type: kb
+---
+
+## Environment
+<table>
+	<tr>
+		<td>Product</td>
+		<td>RadGridView for WPF</td>
+	</tr>
+</table>
+
+## Description
+
+How to increase/decrease the period that a tooltip for a GridViewCell is visible.
+
+## Solution
+
+You can change the tooltip duration by setting the __ToolTipService.ShowDuration__ attached property of the cells via a style.
+
+#### __[XAML] Example 1: Setting ToolTipService.ShowDuration__
+{{region xaml-kb-gridview-howto-change-cells-tooltip-duration-0}}
+
+    <!-- If you're using the NoXaml binaries, you need to add the BasedOn="{StaticResource GridViewCellStyle}" attribute -->
+    <Style TargetType="telerik:GridViewCell">
```

Comment:
Show where this style is defined. This won't be obvious for most WPF newbies. For example, you can wrap the Style into Window.Resources tag.